jlm/mlir/frontend/MlirToJlmConverter.cpp

```diff
@@ -690,10 +690,6 @@ MlirToJlmConverter::ConvertOperation(
         std::vector(inputs.begin(), inputs.end()),
         ConvertType(type)));
   }
-  else if (auto MallocOp = ::mlir::dyn_cast<::mlir::jlm::Malloc>(&mlirOperation))
-  {
-    return jlm::llvm::MallocOperation::create(inputs[0]);
-  }
```

Comment:
Why was this removed?

Reply:
Because there is another one five lines above it, that is identical. This if is thus dead.

I wasn't originally intending to include it in this PR, I staged it by accident.